### PR TITLE
[travis] Add sonarcloud addon

### PR DESCRIPTION
**Pull request by argenos**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,21 +16,38 @@ before_install:
...
+
+addons:
+  sonarcloud:
+    organization: "b-it-bots"
+    token:
+      secure: "dRKXZht+QEdakRytXcHQQ2BeCbc6aJnKgsJoRfew90Os6jwYHsblZxgd6hT339wks8/sIq7QEQHj7Mk7h/WL9TCbkQuSmrK9sNLV0O8RYFtvxPJ7XLHM9eV4IkJSijTxfoY3KiKi/C70tWnMs8fNLVRAm68WqzHwyF0DTtr6bZU2P3YeGs8Vv559j2Rh7kXp4k7l5P9ZTo6HE5vOpt307wsNBNZLiJTSjA6RQUOUlr+LxwT7bTgiIakIbmZ7kLP+dAlF8Dwc00dnBGzOeDJ0Gocy8aoxwGKxyllOVgs3F4+7fUKSf0J7qP3JM92xv+CeKgOu/L/ktAnDwpwat0Xoweo4I1sIhaR1mCDBbHNNS8WnfVX3ItqL0F7n/6+BAn76BuFMnjVzqYHXNwLQL9H5sJhj7OMX5apOEsBcxDErgDA/umeWjvccgjqd79yCfVkq5zHueBeTEDAzfrQNyLypusZZSjfAkrbtmxK6KhWXuz4kmyR20sLNbTZiC6RgVAsSpVaJt8qpqac2bG7dqj/8vx8sso8VLRFmmPGXuF4UQI/ZYb7yWpb4/fU3kNTi+sFyNDpZe4zQITSnWGtplpS7gyejY/mMj6yQ2lElTZAvQXqeuWr5N08BB8oDRqxFQJjXae18yOoWvsDKSw0nSou7E3qHA6aqKGnCCU8FvPbv0hA="
```
Comment (PatrickNa) [inline .travis.yml:53]: Are the tokens visible to everyone?

Comment (argenos) [reply .travis.yml:53]: They're encrypted and that's only the public key. Travis has the private key, see [here](https://docs.travis-ci.com/user/encryption-keys/)

Comment (PatrickNa) [reply .travis.yml:53]: Yes. I read about that, but was not sure how they appear. Good then! Thanks!

**Review by PatrickNa — APPROVED** [review]

**Comment by argenos** [comment]
I've added [stale bot](https://probot.github.io/apps/stale/) to help us manage old issues.

**Comment by PatrickNa** [comment]
> I've added [stale bot](https://probot.github.io/apps/stale/) to help us manage old issues.

Can you give a crash course at some point?

**Comment by argenos** [comment]
It's just a bot that will comment on the issues after 3 months of no activity.

**Review by alex-mitrevski — APPROVED** [review]

**Pull request merged**